src/api/lib/backend/api/cloud.rb

```diff
@@ -10,6 +10,10 @@ def self.upload(user, params, target_name = 'ec2')
         post(['/cloudupload'], params: params.merge(user: user.login, target: target_name), data: data)
       end
 
+      def self.status(user)
```

Comment:
I see 3 problems here:

* This method lacks of a comment for explaining what it does.
* Also it have the same name as the next method in this file.
* And the `to_query` isn't needed, you need to expand it like it is done at https://github.com/openSUSE/open-build-service/blob/master/src/api/lib/backend/api/build_results/binaries.rb#L54, so you only need to do it like this:
```ruby
get(['/cloudupload'], params: { name: user.upload_jobs.pluck(:id) }, expand: [:name])
```

Reply:
Agreed on the first two points. Regarding your last point: Is this somewhere documented how to use the backend lib?